tool/site/client-react/src/components/claim/claimPage.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react"
+import {useEffect, useState} from "react"
+import {ErrorMessage, Label} from "nhsuk-react-components"
+import ClaimForm, {ClaimFormValues, StaticProductInfo} from "./claimForm"
+import axios from "axios"
+import {
+  getMedicationDispenseResources,
+  getMedicationRequestResources,
+  getPatientResources
+} from "../../fhir/bundleResourceFinder"
+import * as fhir from "fhir/r4"
+import Pre from "../pre"
+import {createClaim, getMedicationDispenseLineItemId} from "./createDispenseClaim"
+import {getTaskBusinessStatusExtension} from "../../fhir/customExtensions"
+
+interface ClaimPageProps {
+  baseUrl: string
+  prescriptionId: string
+}
+
+const ClaimPage: React.FC<ClaimPageProps> = ({
+  baseUrl,
+  prescriptionId
```

Comment:
do we want to look at any potential values we want to handle in context at this point, before we add any more components?

Reply:
I've not come across anything where I thought we needed to use context yet, but yeah let's talk about it